mono/metadata/icall-def.h

```diff
@@ -1064,7 +1064,8 @@ ICALL(ILOCK_18, "Exchange(object&,object)", ves_icall_System_Threading_Interlock
 NOHANDLES(ICALL(ILOCK_19, "Exchange(single&,single)", ves_icall_System_Threading_Interlocked_Exchange_Single))
 NOHANDLES(ICALL(ILOCK_20, "Increment(int&)", ves_icall_System_Threading_Interlocked_Increment_Int))
 NOHANDLES(ICALL(ILOCK_21, "Increment(long&)", ves_icall_System_Threading_Interlocked_Increment_Long))
-NOHANDLES(ICALL(ILOCK_22, "Read(long&)", ves_icall_System_Threading_Interlocked_Read_Long))
+NOHANDLES(ICALL(ILOCK_22, "MemoryBarrierProcessWide", ves_icall_System_Threading_Interlocked_MemoryBarrierProcessWide))
```

Comment:
You need to bump monolite version, see https://github.com/mono/mono/blob/77e99d080d9c4419c546f0ba1023bf5653ac2dbd/configure.ac#L62

Reply:
Done